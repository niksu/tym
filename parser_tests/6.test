v(a).
v(b).
v(c).
e(a,a).
e(b,b).
e(a,b).
e(b,c).
e(X,Y) :- e(X, Z), e(Z, Y).

%?- e(a,c).
