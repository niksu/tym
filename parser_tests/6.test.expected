input contents |v(a).
v(b).
v(c).
e(a,a).
e(b,b).
e(a,b).
e(b,c).
e(X,Y) :- e(X, Z), e(Z, Y).

%?- e(a,c).
|
(no query given)
stringed file contents (size=282, remaining=2718)
|predicate_v(constant_a).
predicate_v(constant_b).
predicate_v(constant_c).
predicate_e(constant_a, constant_a).
predicate_e(constant_b, constant_b).
predicate_e(constant_a, constant_b).
predicate_e(constant_b, constant_c).
predicate_e(X, Y) :- predicate_e(X, Z), predicate_e(Z, Y).|
stringed file contents strlen=281
